src/Console/Output/ErrorOutput.php

```diff
@@ -117,16 +117,27 @@ public function listErrors(string $process, array $errors): void
         }
     }
 
+    /**
+     * @param array{
+     *     function: string,
+     *     line?: int,
+     *     file?: string,
+     *     class?: class-string,
+     *     type?: '::'|'->',
+     *     args?: mixed[],
+     *     object?: object,
+     * } $trace
+     */
     private function outputTrace(array $trace): void
     {
-        if (isset($trace['class'], $trace['type'], $trace['function'])) {
```

Comment:
we don't know that `function` is actually present.
yes, you added a docblock, but compiler is fully ignoring it in runtime - it's useful only fo SCA tools.

Reply:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6633

Reply:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6633#issuecomment-1246034206